src/content/docs/guides/prerequisites/index.mdx

```diff
@@ -104,7 +104,45 @@ sudo zypper in -t pattern devel_basis
   </TabItem>
   <TabItem label="NixOS">
 
-TODO: Need to build out NixOS instructions
+`flake.nix`:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { nixpkgs, rust-overlay, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        overlays = [ (import rust-overlay) ];
+        pkgs = import nixpkgs { inherit system overlays; };
+      in {
+        devShells.default = pkgs.mkShell {
+          nativeBuildInputs = [ pkgs.pkg-config ];
+          buildInputs = [
+            (pkgs.rust-bin.fromRustupToolchain {
+              channel = "nightly-2023-12-19"; # feel free to change the channel
+              components = [ "rustfmt" "rust-src" ];
+              targets = [ "wasm32-unknown-unknown" ];
+              profile = "minimal";
+            })
+            pkgs.pkg-config
+            pkgs.atk
+            pkgs.pango
+            pkgs.libsoup_3
+            pkgs.webkitgtk_4_1
+            pkgs.openssl
+            pkgs.cargo-tauri
+            pkgs.trunk
```

Comment:
currently fails because of missing `libudev.pc`, which is included in systemd
```suggestion
            pkgs.trunk
            pkgs.systemdLibs
```

Reply:
This one might actually be unrelated to Tauri itself but the project I've tested it with. 
Need confirmation